addons/base_import_module/models/ir_module.py

```diff
@@ -392,6 +411,16 @@ def _get_missing_dependencies(self, zip_data):
                 description += "- " + mod.shortdesc + "\n"
         return description
 
+    @api.model
+    def search_panel_select_range(self, field_name, **kwargs):
+        if field_name == 'category_id' and any(dom[0] == 'module_type' and dom[-1] == 'industries' for dom in kwargs.get('category_domain', [])):
```

Comment:
shouldn't we check the operator too?
```suggestion
        if field_name == 'category_id' and any(tuple(dom) == ('module_type', '=', 'industries') for dom in kwargs.get('category_domain', [])):
```